### ISSUE TITLE
Park name not redering on standard layer

### ISSUE THREAD

**Issue by nevw** [gravitystorm/openstreetmap-carto#4997]
Expected the park name to render https://www.openstreetmap.org/relation/17888368

Park name renders on other layers at osm.org but not on the standard layer

What is the reason?

**Comment by imagico** [comment]
The reason is #3880.  The label for the park is blocked by the symbols for the POIs within the park.  Displaying these properly would require combining symbols and labels in one layer and sorting by starting zoom level.

Closing as duplicate of #3880.

**Comment by nevw** [comment]
Thanks for the info re the name and poi symbol clash
